### DISCUSSION TITLE
customasm not working

### DISCUSSION THREAD

**Discussion by psyhigh** [hlorenzi/customasm#163] [Q&A]
hi i tried numorus versions and non of them work the windos shows for les than a second and nothing i havr win 7 vs studio 2022 and vs code 2020.  help

**Comment by hlorenzi** [comment]
The app works via the command line. You've got to use it through a terminal app, such as PowerShell, or the embedded terminal in VSCode. Have you tried the instructions [from this page](https://github.com/hlorenzi/customasm/wiki/Getting-started) yet?
